common/src/main/java/com/wynntils/wc/utils/scoreboard/ScoreboardManager.java

```diff
@@ -140,8 +155,53 @@ public class ScoreboardManager {
                 }
             }
         }
+
+        handleScoreboardReconstruction();
     };
 
+    private static void handleScoreboardReconstruction() {
```

Comment:
How would this work if the user logs on to another server? I think one of the (less important) design goals of Artemis is that the mod should be "transparent" on other servers, so you should in principle be able to have it always enabled, even if playing on other servers.

Is there some logic that skips this process if not on Wynncraft?

Reply:
Found it now in onSetScore